MFSDA/MFSDA.py

```diff
@@ -283,81 +274,8 @@ def onCSVFile(self):
         self.pushButton_run.connect('clicked(bool)', self.onKillComputation)
 
         return
- 
-        '''        self.param = {}
-        self.param["shape"] = self.lineEdit_pshape.currentPath
-        self.param["pvalues"] = self.lineEdit_output.directory+'/pvalues.json'
-        self.param["efit"] = self.lineEdit_output.directory+'/efit.json'
-        self.param["covariates"] = self.lineEdit_CovariateNames.currentPath
-        self.param["output"] = self.lineEdit_output.directory+'/out.vtk'
-
-        MFSDAShapemodule = slicer.modules.MFSDA_createShapes
-        self.MFSDAShapeThread=slicer.cli.run(MFSDAShapemodule, None, self.param, wait_for_completion=False)
-
-        return
-
-        CSVFile=open(self.lineEdit_output.directory+'/output.csv', 'wb')
-        Data=['VTK Files']
-        with CSVFile:
-            writer = csv.writer(CSVFile)
-            writer.writerow(Data)
-            writer.writerow([self.lineEdit_pshape.currentPath])
-        CSVFile.close()
-        csvFilePath = self.lineEdit_output.directory+'/output.csv'
-        slicer.modules.shapepopulationviewer.widgetRepresentation().loadCSVFile(csvFilePath)
-        slicer.util.selectModule(slicer.modules.shapepopulationviewer)
-
-        return
-
-
-        args=arguments(coordData=self.lineEdit_pshape.currentPath, covariate=self.lineEdit_covariate.currentPath, covariateType=self.lineEdit_covariateType.currentPath, outputDir=self.lineEdit_output.directory, shapeData=self.lineEdit_csv.currentPath, shapePath=PathOutput)
-        self.logic.run_script(args)
-
-
-        pvaluesPath=self.lineEdit_output.directory+'/pvalues.json'
```

Comment:
@jcfr this is not commented code, is this no longer needed?

Reply:
If you look at the top and bottom of the snippet, you can see it is surrounded by `'''`